README.md

```diff
@@ -1,3 +1,148 @@
+
+The simplest example to send a Email is:
+
+```csharp
+using var httpClient = new HttpClient();
+using var tokenProviderFactory = new LogicTokenProviderFactory(configuration.TokenProvider);
+var emailClient = new EmailClient(httpClient, tokenProviderFactory, configuration.EmailOptions);
+
+var sendEmailRequest = new SendEmailRequestDetails(
+    providerConfigurationId: configuration.ProviderConfigurationId,
+    recipients: new RecipientEmailDetails(
+            [
+                {
+                    email = " To Email Addres 1"
+                },
+                {
+                    email = "To Email Addres 2"
+                }
+            ],
+            [
+                {
+                    email = " Cc Email Addres 1"
+                },
+                {
+                    email = "Cc Email Addres 2"
+                }
+            ],
+            [
+                {
+                    email = "Bcc Email Addres 1"
+                },
+                {
+                    email = "Bcc Email Addres 2"
+                }
+            ]),
+    body: "Hello, world!",
+    subject: "Email subject",
+    attachment: new AttachmentDetails  [
+                {
+                    attachmentId: new Guid("00000000-0000-0000-0000-000000000000")
+                }
+            ],
+    schedule: null,
+    template: new TemplateData(
+    {
+        templateId: new Guid("00000000-0000-0000-0000-000000000000"),
+        mergeData: {}
+    }
+    ),
+    callbackUrl: "provide your callback Url")
+
+var emailResponse = await emailClient.SendEmail(sendEmailRequest).ConfigureAwait(false);
+
+```
+The simplest example for upload attachment is:
+
+```csharp
+using var httpClient = new HttpClient();
+using var tokenProviderFactory = new LogicTokenProviderFactory(configuration.TokenProvider);
+var emailClient = new EmailClient(httpClient, tokenProviderFactory, configuration.EmailOptions);
+
+ var attachmentRequest = new AttachmentRequestDetails(new Guid(configurationId), attachmentFile);
+result = await emailClient.AddAttachment(attachmentRequest).ConfigureAwait(false)
+```
+
+The simplest example for upload template is:
+
+```csharp
+using var httpClient = new HttpClient();
+using var tokenProviderFactory = new LogicTokenProviderFactory(configuration.TokenProvider);
+var emailClient = new EmailClient(httpClient, tokenProviderFactory, configuration.EmailOptions);
+
+ var attachmentRequest = new TemplateRequestDetails(new Guid(configurationId), templateFile);
+result = await emailClient.AddAttachment(attachmentRequest).ConfigureAwait(false)
+```
+
+## Sample application
+## How to configure the Email client
+
+Perhaps the easiest way to configure the Email client is from Application Settings.
+
+```json
+{
+  "TokenProvider": {
+    "ClientId": "",
+    "ClientSecret": "",
+    "AuthorizationScope": ""
+  },
+  "EmailOptions": {
+    "SubscriptionId": ""
+  }
+}
+```
+
```

Comment:
Instead of having repeated phrases below for all samples, you can just a use a header `Samples` and then 3 points under it as `Upload Template`, `Upload Attachment` and `Send Email` and use hyperlinks on each of these points to navigate to respective sample folder.

Reply:
Changed as suggested :)